### PR TITLE
06 get reviews id comments

### PR DESCRIPTION
**Pull request by MikeDobinson**
not fully happy with error handling. GET api/reviews/999/comments will return comments not found, NOT no review by that ID

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -77,6 +77,38 @@ describe('/api/reviews', () => {
...
+          .then(({ body }) => {
+            const { comments } = body;
+            expect(Array.isArray(comments)).toBe(true);
+            expect(comments.length).toBe(3);
+            expect(comments).toBeSortedBy('created_at', { descending: true });
+            comments.forEach((comment) => {
+              expect(comment).toMatchObject({
+                comment_id: expect.any(Number),
+                body: expect.any(String),
+                review_id: expect.any(Number),
```
Comment (roisinoreilly) [inline __tests__/app.test.js]: we know that the review id should be 2 here not any number

File: __tests__/app.test.js [hunk 2]
```diff
@@ -77,6 +77,38 @@ describe('/api/reviews', () => {
...
+                votes: expect.any(Number),
+                created_at: expect.any(String),
+              });
+            });
+          });
+      });
+      it('returns 404: no comments found if no comments assigned to an existing review ', () => {
+        return request(app)
+          .get('/api/reviews/10/comments')
+          .expect(404)
```
Comment (roisinoreilly) [inline __tests__/app.test.js]: this is actually a 200, it's a valid request to a valid endpoint, it should instead return an empty array of comments

**Review by roisinoreilly — CHANGES_REQUESTED** [review]
great job so far! There's just a few more tests I'd like you to implement, a valid id with no comments (a 200), an invalid id (eg. /not-an-id/comments) and a valid but non existent id (eg. /9999/comments). After you've added those send us another PR :)

**Review by alexdevdavis — APPROVED** [review]
Nicely done! Happy to approve this 👍

**Pull request merged**